components/Slider/Slider.css

```diff
@@ -0,0 +1,42 @@
+.root {
+  appearance: none;
+  background-color: var(--color-grey-lighter);
+  border: 1px solid var(--color-grey-lighter);
+  height: 0.5em;
+  border-radius: 1em;
+  cursor: pointer;
+  width: 100%;
+}
+
+.root:focus {
+  outline: none;
+}
+
+.root::-webkit-slider-thumb {
```

Comment:
What does this look like on Firefox or IE?

We're using autoprefixer - so if these rules are based on a standard we should be able to do without the vendor prefixes.